### PR TITLE
feat(api): add search all workflows api + add docker layer in buildinfo

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20444.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

File: engine/sql/api/295_v2_run_git_context.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
```
Comment (yesnault) [inline engine/sql/api/295_v2_run_git_context.sql:1]: It's 296 now

**Review by yesnault — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20447.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20447.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**